service-commands/scripts/userClockValues.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * Find the creator nodes associated with a user and their respective clock values.
+ *
+ * Script usage: node userClockValues.js <size> <outFile>
```

Comment:
can we spit out usage examples and update this comment if it hasn't been yet?

Reply:
like how does this relate to the expected discoveryProviderEndpoint? 

https://github.com/AudiusProject/audius-protocol/pull/1121/files#diff-c1d426348d148ea1ceeaaa10854aab61d9d22be34a5294b896c55e31b43f9a15R151

Reply:
Thanks, that was stale code, updated 👍